owncloudApp/src/main/java/com/owncloud/android/extensions/ActivityExt.kt

```diff
@@ -53,10 +53,13 @@ fun Activity.showErrorInToast(
         ).show()
     }
 
-fun Activity.goToUrl(url: String) {
+fun Activity.goToUrl(
+    url: String,
+    flags: Int = 0) {
     if (url.isNotEmpty()) {
         val uriUrl = Uri.parse(url)
         val intent = Intent(Intent.ACTION_VIEW, uriUrl)
+        if (flags != 0) intent.addFlags(flags)
         startActivity(intent)
```

Comment:
Reformat this code, and I think that instead of using 0 as the default value, we could make it nullable and add flags only when we receive actual flags. What do you think?